lib/medic/store.rb

```diff
@@ -113,5 +115,32 @@ def disable_all_background_delivery(block=Proc.new)
       })
     end
 
+  private
+
+    def prepare_for_save(sample)
```

Comment:
This method will get even bigger when we add workout support and that shorthand I was talking about. I'm open to any ideas for cleaning this up.